quickjs.c

```diff
@@ -23770,9 +23266,7 @@ static __exception int js_parse_assign_expr2(JSParseState *s, int parse_flags)
             static const uint8_t assign_opcodes[] = {
                 OP_mul, OP_div, OP_mod, OP_add, OP_sub,
                 OP_shl, OP_sar, OP_shr, OP_and, OP_xor, OP_or,
-#ifdef CONFIG_BIGNUM
```

Comment:
Is it ok for op_pow to be repeated?

Reply:
It looked very deliberate so I left it in but see the new commit, I believe this array can be removed altogether with some careful reshuffling. It's rather subtle though so I would appreciate it if you looked it over carefully.

Reply:
I don't think I can make sense of that part... so I trust you, and the tests :-)

Reply:
I tried to make sense of it 😅 ... does the enum which contain TOK_MUL_ASSIGN need to be in sync with the opcodes? Then again, if it was wrong some test would have failed right?

Reply:
> does the enum which contain TOK_MUL_ASSIGN need to be in sync with the opcodes?

Yes. That array was used to map from the TOK domain to the OP domain. It becomes superfluous when both have the same ordering because then you can simply compute the right value.